config/initializers/omniauth.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :twitter, ENV['twitter_key'], ENV['twitter_secret']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.